multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/util/ClientReleaser.java

```diff
@@ -27,10 +27,13 @@ public void releaseClientFor(HistoryService historyService, String processInstan
         String organizationName = HistoryUtil.getVariableValue(historyService, processInstanceId, Variables.ORGANIZATION_NAME.getName());
         String spaceName = HistoryUtil.getVariableValue(historyService, processInstanceId, Variables.SPACE_NAME.getName());
         String spaceGuid = HistoryUtil.getVariableValue(historyService, processInstanceId, Variables.SPACE_GUID.getName());
+        String correlationId = HistoryUtil.getVariableValue(historyService, processInstanceId, Variables.CORRELATION_ID.getName());
 
         try {
-            clientProvider.releaseClient(user, organizationName, spaceName);
-            clientProvider.releaseClient(user, spaceGuid);
+            clientProvider.releaseClient(user, organizationName, spaceName, null);
```

Comment:
I think, that correlationid should be not null at this point

Reply:
It can be null if the request to CC is not performed from process i.e. `cf mtas`.